rsconnect/api.py

```diff
@@ -406,14 +509,30 @@ def validate_server(
             return self
 
         # If our info came from the command line, make sure the key really works.
-        if not from_store:
-            _ = self.verify_api_key()
+        if not server_data.from_store:
+            _ = self.verify_api_key(connect_server)
 
-        self.connect_server = connect_server
-        self.client = RSConnect(self.connect_server)
+        self.remote_server = connect_server
+        self.client = RSConnectClient(self.remote_server)
 
         return self
 
+    def validate_shinyapps_server(
+        self, url: str = None, account_name: str = None, token: str = None, secret: str = None, **kwargs
```

Comment:
The format for the arguments here is different from the above - is black supposed to reformat these?

Reply:
This is how black formatted it. I'm guessing it tries to fit the parameters all onto a single line, but if that's too long, it would break them up onto separate lines.